ansible_3par_docker_plugin/properties/plugin_configuration_properties_sample.yml

```diff
@@ -40,6 +40,20 @@ INVENTORY:
     #  hpe3par_password: 3pardata
     #  vlan_tag: False
 
+#Mandatory Parameters for Filepersona-----------------------------------------------------------------------------------
+  DEFAULT_FILE:
+    # Specify the port to be used by HPE 3PAR plugin etcd cluster
+    host_etcd_port_number: 23790
+    # Plugin Driver - File driver
+    hpedockerplugin_driver: hpedockerplugin.hpe.hpe_3par_file.HPE3PARFileDriver
+    hpe3par_ip: 192.168.2.50
+    hpe3par_username: 3paradm
```

Comment:
Please refrain from using actual username, password, cpg-name etc in any of the sample files going forward.
Give dummy values instead.